rocketpy/rocket/rocket.py

```diff
@@ -1161,6 +1161,7 @@ def add_air_brakes(
         sampling_rate,
         clamp=True,
         reference_area=None,
+        initial_observed_variables=None,
         name="AirBrakes",
         controller_name="AirBrakes Controller",
     ):
```

Comment:
Regarding the `plots` situation:

- The earlier implementation has the benefit that we control the `observed_variables`, therefore we could make useful plots;
- In this PRs case, we don't have this information due to the generality of the controller. However this allows for a much greater flexibility.

In my opinion, it would be interesting to have the best of both worlds by using the "convenience" methods for controller creation - such as `add_air_brakes` - to obtain enough information for generating the plots.

In general, one looking for a complex control will generate it from scratch and make their own plots. Alternatively, one using the convenience of `add_air_brakes` likely also wants plots from it without too many manual operations.

Therefore I was looking for a way that we can use the context of `add_air_brakes` to generate their plots. This would require imposing the three first parameters of observed variables and mapping the controller to the air brake somehow, but I could not find a reasonable way to do it.

This was just to explore ways to make this implementation easier for the user, not to pinpoint any particular change. I am open to suggestions.

Reply:
We can just add the state history for the air brakes again, except that the state history would only save information, and none be passed to the controller function. This might still require adding things in the flight class

However, I generally think that if we do a good job with the documentation, the user should be able to easily plot the data and so on

Reply:
Good point. Let's focus on making a proper documentation for this and, if there is the need, we can implement something to plot it automatically.